hwsecurity/fido2/src/main/java/de/cotech/hw/fido2/domain/create/AuthenticatorData.java

```diff
@@ -69,7 +70,7 @@ public abstract class AuthenticatorData {
     // extensions 	variable (if present) 	Extension-defined authenticator data. This is a CBOR [RFC7049] map with extension identifiers as keys, and authenticator extension outputs as values. See §9 WebAuthn Extensions for details.
     @Nullable
     @SuppressWarnings("mutable")
-    public abstract byte[] extensions();
+    public abstract Map extensions();
```

Comment:
It's not useful to have the extensions type (never actually exposed in app-visible objects) be a byte array because some of its contents depend on authenticator state objects like the shared secret :-(.